Commonalities/documentation/SupportingDocuments/Camara_Versioning_Guidelines.md

```diff
@@ -0,0 +1,20 @@
+# Camara versioning guidelines
+
+* Every release includes a **CHANGELOG.md** file. Please make sure that the content is structured in a format that is easy to read.
```

Comment:
It would be good to agree on a common location for this file, for example code/API_definitions

Reply:
@jlurien CHANGELOG.md is normally at the root of the repo alongside README.md

Reply:
Yes, maybe instead of having a CHANGELOG.md for each realease is more convenient actually, to have a common file in `main` branch included all cumulative changes of each release/version i.e. actually each new release con just change that file adding the corresponding changes.

Reply:
@jpengar CHANGELOG.md will be made available in every release branch. It's location is right beside the README file. In the main branch the changelog file could either be kept empty or can be used to give an overview as you have stated above of the releases.